src/main/resources/templates/start.html

```diff
@@ -29,12 +29,12 @@ <h2>Collections</h2>
     </div>
     <!-- Items of a selected collection section-->
     <div class="collection-div">
-        <div class="container item-div">
+        <div class="container pt-3 pb-3">
             <div class="row row-cols-auto">
                 <h2>Browse Collection</h2>
-                <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addItemModal">Add Item</button>
+                <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#addItemModal">Add Item</button>
             </div>
-                <div class="row row-cols-auto">
+                <div class="row row-cols-auto pt-1">
```

Comment:
Same as above, this button had no spacing so I made it so there is some space between the header and the UI cards.

Citation: https://getbootstrap.com/docs/4.0/utilities/spacing/